frontend/containers/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -74,9 +75,14 @@ class DocumentEditToolbar extends Component {
   constructor(props) {
     super(props)
 
+    this.keyboard = new Keyboard()
     this.onSave = null
   }
 
+  componentDidMount() {
+    this.keyboard.on('cmd+s').do(this.saveDocument.bind(this))
```

Comment:
I've just changed to what I had proposed and it worked fine for me. Could you pull and test, please?